EXOSIMS/PlanetPhysicalModel/ForecasterMod.py

```diff
@@ -60,7 +60,7 @@ def calc_radius_from_mass(self, Mp):
                 Planet mass in units of Earth mass
 
         Returns:
-            astropy Quantity array:
+            Rp (astropy Quantity array):
```

Comment:
Please revert this to how it used to be.  Returns must specify only the type (no var names at the top level).  When returning multiple values, the type is `tuple` and then we typically list the contents by name.  In this case, with a single return, the first line of Returns should be only the  return type.

Reply:
Since you're changing these - they should be `~astropy.units.Quantity(~numpy.ndarray(float))`